src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/DirectWriteForwarder.vcxproj

```diff
@@ -29,6 +29,7 @@
   <PropertyGroup>
     <UseDestinationLibFolder>true</UseDestinationLibFolder>
     <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
+    <TargetFrameworkIdentifier Condition="$(TargetFramework.StartsWith('net4'))!='true'">.NETFramework</TargetFrameworkIdentifier>
```

Comment:
I'm not sure - does VCXProj support multi-targeting by framework, or will something need to be done with build traversal projects?